### PR TITLE
Add osu!mania pp committee members

### PR DESCRIPTION
**Pull request by apollo-dw**
See https://github.com/ppy/osu/pull/17913#issuecomment-1106540581

Added members and also specified that the panel was osu!. Will need assistance here for updating translations (should be straightforward), or can just outdate them.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/People/Performance_Points_Committee/en.md [hunk 1]
```diff
@@ -7,16 +7,29 @@ tags:
...
+
 - ![][flag_GB] [Apo11o](https://osu.ppy.sh/users/9558549)
 - ![][flag_HU] [emu1337](https://osu.ppy.sh/users/2185987)
 - ![][flag_AU] [MBmasher](https://osu.ppy.sh/users/4498616)
 - ![][flag_RU] [StanR](https://osu.ppy.sh/users/7217455)
 
+### osu!mania
+
+- ![][flag_US] [abraker](https://osu.ppy.sh/users/4635891)
+- ![][flag_ES] [Evening](https://osu.ppy.sh/users/2193881)
```
Comment (Eve-ning) [inline wiki/People/Performance_Points_Committee/en.md:24]:
```suggestion
- ![][flag_SG] [Evening](https://osu.ppy.sh/users/2193881)
```
I think you meant `SG`

**Comment by Walavouchey** [comment]
mind also writing a bit about how and why the mania pp committee was formed? outdating the translations is ok

**Comment by apollo-dw** [comment]
the linked post in the PR description is pretty much all there is to it i'm afraid: a change was proposed and they were like "ok its time to form one" and then one was made. do you think it's a good idea to write something anyways?

**Comment by Walavouchey** [comment]
yes

File: wiki/People/Performance_Points_Committee/en.md [hunk 2]
```diff
@@ -7,16 +7,30 @@ tags:
...
 
-The panel was first announced to the public through the [November 2021 *Performance Points & Star Rating Updates* news post](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) in response to a [community survey](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) held earlier that year.
+The osu! panel was first announced to the public through the [November 2021 *Performance Points & Star Rating Updates* news post](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) in response to a [community survey](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) held earlier that year.
+
+osu!mania's panel was formed on the 22nd April 2022 in anticipation of a surge in active development effort, starting with [a change that fixed critical issues in osu!mania's strain skill](https://github.com/ppy/osu/pull/17913).
```
Comment (Walavouchey) [inline wiki/People/Performance_Points_Committee/en.md:12]:
```suggestion
osu!mania's panel was formed 22 April 2022 in anticipation of a surge in active development effort, starting with [a change in difficulty calculation for hold note releases in osu!mania](https://github.com/ppy/osu/pull/17913).
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**